storage/backend/ceph/ceph.go

```diff
@@ -41,6 +42,21 @@ type Driver struct {
 	mountpath string
 }
 
+func (c *Driver) mkMountPath(poolName, intName string) (string, error) {
+	// Directory to mount the volume
+	volumePath := filepath.Join(c.mountpath, mkpool(poolName, intName))
+	rel, err := filepath.Rel(c.mountpath, volumePath)
+	if err != nil || strings.Contains(rel, "..") {
+		return "", errors.MountFailed.Combine(errored.Errorf("Calculated volume path would escape subdir jail: %v", volumePath))
```

Comment:
I'll look into this, we need the ceph device to exhibit this behavior.

I'm also not 100% certain it matters because we'll be overwriting the dir with the mounted content anyway.